container/shim/package.json

```diff
@@ -35,7 +37,9 @@
     "eslint-plugin-jsdoc": "^39.3.3",
     "eslint-plugin-promise": "^6.0.0",
     "husky": "^8.0.1",
-    "nodemon": "^2.0.18"
+    "nock": "^13.2.9",
+    "nodemon": "^2.0.18",
+    "test": "^3.2.1"
```

Comment:
Any reason not to use native test? https://nodejs.org/api/test.html

Reply:
Nvm, still experimental, and Node 18+ only

Reply:
that's it 👍